Source/WebCore/rendering/svg/legacy/LegacyRenderSVGShape.cpp

```diff
@@ -107,7 +107,7 @@ bool LegacyRenderSVGShape::shapeDependentStrokeContains(const FloatPoint& point,
 
 bool LegacyRenderSVGShape::shapeDependentFillContains(const FloatPoint& point, const WindRule fillRule) const
 {
-    return path().contains(point, fillRule);
+    return hasPath() && path().contains(point, fillRule);
```

Comment:
I think this can be fixed like this:
```
bool LegacyRenderSVGRect::shapeDependentFillContains(const FloatPoint& point, const WindRule fillRule) const
{
    if (m_shapeType == ShapeType::Empty)
        return false;
    if (m_shapeType != ShapeType::Rectangle)
        return LegacyRenderSVGShape::shapeDependentFillContains(point, fillRule);
    return m_fillBoundingBox.contains(point.x(), point.y());
}
```

Reply:
Also you need to do the same fix in LBSE renderers `RenderSVGRect` or `RenderSVGShape`.